src/Pages/JobseekerData.js

```diff
@@ -0,0 +1,180 @@
+    // console.log(jobseeker);
+  };
+
+  // TODO: saving the state for skills and education
+
+  return (
+    <div>
+      <div>
+        [Image goes here]
+      </div>
+      <div className="content">
+        <div>
+          <h1>Employment Roadmap</h1>
+        </div>
+        <div>
+          <h1>Client Info</h1>
+          <form>
+            <div className="inputWrapper">
+              <label htmlFor="authName">
+                Authentic Name
+                <input
+                  id="authName"
+                  placeholder={jobseeker.name}
+                  onChange={(e) => {
+                    setJobseeker({
+                      ...jobseeker,
+                      name: e.target.value,
+                    });
+                  }}
+                  type="text"
+                />
+              </label>
+            </div>
+            <div className="inputWrapper">
+              <label htmlFor="pronouns">
+                Pronouns
+                <input
+                  id="pronouns"
+                  placeholder={jobseeker.pronouns}
+                  onChange={(e) => {
+                    setJobseeker({
+                      ...jobseeker,
+                      pronouns: e.target.value,
+                    });
+                  }}
+                  type="text"
+                />
+              </label>
+            </div>
+            <div className="inputWrapper">
+              <label htmlFor="phone">
+                Phone
+                <input
+                  id="phone"
+                  placeholder={jobseeker.phone}
+                  onChange={(e) => {
+                    setJobseeker({
+                      ...jobseeker,
+                      phone: e.target.value,
+                    });
+                  }}
+                  type="text"
+                />
+              </label>
+            </div>
+            <div className="inputWrapper">
+              <label htmlFor="email">
+                Email
+                <input
+                  readOnly // because email is the docid, so crucial that you cannot change it.
+                  id="email"
+                  placeholder={jobseeker.email}
+                  type="text"
+                />
+              </label>
+            </div>
+            {/* <div className="inputWrapper">
+                        <label for="location">City/State</label>
+                        <input id="location" placeholder="Kevin" type="text"></input>
+                    </div> */}
+            <div className="inputWrapper">
+              <label htmlFor="ethnicity">
+                Ethnicity
+                <input
+                  id="ethnicity"
+                  placeholder={jobseeker.ethnicity}
+                  onChange={(e) => {
+                    setJobseeker({
+                      ...jobseeker,
+                      ethnicity: e.target.value,
+                    });
+                  }}
```

Comment:
Also one more thing—I know we talked to the desginers about basically not creating an account in the db until the assement has been completed. It would be great to figure out a way that we can delete account entries in the db if the user hasn't completed the assessment!